### ISSUE TITLE
Uncaught Callback must be a function

### ISSUE THREAD

**Issue by 0xLyptox** [brhaka/sfml-compiler#2]
**Atom**: 1.41.0-nightly1 x64
**Electron**: 4.2.7
**OS**: DESKTOP-0N5MG0H
**Thrown From**: [sfml-compiler](https://github.com/brhaka/sfml-compiler) package 1.3.2

### Stack Trace

Uncaught Callback must be a function

```
At fs.js:129

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at maybeCallback (fs.js:129:9)
    at Object.writeFile (fs.js:1137:14)
    at Object.toggle (/packages/sfml-compiler/lib/sfml-compiler.js:161:7)
    at /packages/sfml-compiler/lib/sfml-compiler.js:90:20)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom-nightly/app-1.41.0-nightly1/resources/app/static/<embedded>:11:349102)
    at KeymapManager.dispatchCommandEvent (~/AppData/Local/atom-nightly/app-1.41.0-nightly1/resources/app/static/<embedded>:11:1249829)
    at KeymapManager.handleKeyboardEvent (~/AppData/Local/atom-nightly/app-1.41.0-nightly1/resources/app/static/<embedded>:11:1245963)
    at WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom-nightly/app-1.41.0-nightly1/resources/app/static/<embedded>:11:284308)
```

### Commands

```
     -8:14.6.0 core:save (input.hidden-input)
     -8:13.5.0 gpp-compiler:compile (input.hidden-input)
  2x -7:34.9.0 core:select-all (input.hidden-input)
     -6:02.6.0 core:paste (input.hidden-input)
     -5:48.4.0 core:save (div.package-detail)
     -5:12.2.0 sfml-compiler:toggle (input.hidden-input)
     -4:41.5.0 core:select-all (input.hidden-input)
     -4:41.2.0 core:backspace (input.hidden-input)
     -4:38.1.0 core:save (input.hidden-input)
     -4:34.6.0 gpp-compiler:compile (input.hidden-input)
     -4:30.8.0 gpp-compiler:gdb (input.hidden-input)
     -4:16.1.0 sfml-compiler:toggle (input.hidden-input)
     -1:12.3.0 core:copy (input.hidden-input)
     -1:11.4.0 core:paste (input.hidden-input)
     -0:48.5.0 core:save (div.package-detail)
     -0:43.8.0 sfml-compiler:toggle (input.hidden-input)
```

### Non-Core Packages

```
gpp-compiler 3.0.7 
intellij-idea-keymap 0.2.3 
sfml-compiler 1.3.2 
```

**Comment by brhaka** [comment]
Looks like this is a problem that is happening on **FS** starting on **Atom 1.39**.

The problem is exactly in *sfml-compiler.js*, line 161: `fs.writeFile(pathToCurrentWorkingDir + "\\compiler.bat", compileCommands);`, and maybe on lines 184 & 186 as well.

Let's wait until the next update or any new issue about that on [Atom's GitHub Repo](https://github.com/atom/atom).

**Comment by Daedluz** [comment]
I have the same problem.
Can someone help me?
Pls


[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Windows 10
**Thrown From**: [sfml-compiler](https://github.com/brhaka/sfml-compiler) package 1.3.4


### Stack Trace

Uncaught Callback must be a function

```
At fs.js:129

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at maybeCallback (fs.js:129:9)
    at Object.writeFile (fs.js:1137:14)
    at Object.compile (/packages/sfml-compiler/lib/sfml-compiler.js:168:7)
    at /packages/sfml-compiler/lib/sfml-compiler.js:90:20)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:349385)
    at KeymapManager.dispatchCommandEvent (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:1252387)
    at KeymapManager.handleKeyboardEvent (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:1248521)
    at WindowEventHandler.handleDocumentKeyEvent (~/AppData/Local/atom/app-1.42.0/resources/app/static/<embedded>:11:284591)
```

### Commands

```
```

### Non-Core Packages

```
gpp-compiler 3.0.7 
sfml-compiler 1.3.4 
```

**Comment by brhaka** [comment]
Fixed by @Kacarott ( #3 )
